osi_lane.proto

```diff
@@ -1068,6 +1068,14 @@ message LaneBoundary
             // A sound barrier.
             //
             TYPE_SOUND_BARRIER = 15;
+
+            // The lane boundary is a fence.
+            //
+            TYPE_FENCE = 16;
```

Comment:
A FENCE could be described with STRUCTURE?

Reply:
At the moment we have mapped STRUCTUR to WALL.